PlayCover/View/AppsView.swift

```diff
@@ -15,15 +15,15 @@ struct AppsView: View {
 
     @EnvironmentObject var appVm: AppsVM
 
-    @State private var gridLayout = [GridItem(.adaptive(minimum: 150, maximum: 150), spacing: 0)]
+    @State private var gridLayout = [GridItem(.adaptive(minimum: 160, maximum: 160), spacing: 0)]
 	@State private var alertTitle = ""
 	@State private var alertText = ""
 	@State private var alertBtn = ""
 	@State private var alertAction : (() -> Void) = {}
 	@State private var showAlert = false
 
     var body: some View {
-        VStack(alignment: .leading, spacing: 0) {
+        VStack(alignment: .center, spacing: 0) {
```

Comment:
Why? If you're trying to centre the app icons themselves, this is the wrong view to centre.

Reply:
This VStack is centered so that the search bar can be centered instead of sticking to the leading side of the window as it gets expanded and repositioned.

Reply:
@IsaacMarovitz I don't know if this still has to be changed. Other than that, the PR is good to go.